packages/components/upload/src/upload.ts

```diff
@@ -79,100 +79,178 @@ export interface UploadHooks {
 }
 
 export const uploadBaseProps = buildProps({
+  /**
+   * @description request URL
+   */
   action: {
     type: String,
     default: '#',
   },
+  /**
+   * @description request headers
+   */
   headers: {
     type: definePropType<Headers | Record<string, any>>(Object),
   },
+  /**
+   * @description set upload request method
+   */
   method: {
     type: String,
     default: 'post',
   },
+  /**
+   * @description additions options of request
+   */
   data: {
     type: Object,
     default: () => mutable({} as const),
   },
+  /**
+   * @description whether uploading multiple files is permitted
+   */
   multiple: {
     type: Boolean,
     default: false,
   },
+  /**
+   * @description key name for uploaded file
+   */
   name: {
     type: String,
     default: 'file',
   },
+  /**
+   * @description whether to activate drag and drop mode
+   */
   drag: {
     type: Boolean,
     default: false,
   },
+  /**
+   * @description whether cookies are sent
+   */
   withCredentials: Boolean,
+  /**
+   * @description whether to show the uploaded file list
+   */
   showFileList: {
     type: Boolean,
     default: true,
   },
+  /**
+   * @description accepted [file types](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attr-accept), will not work when `thumbnail-mode === true`
+   */
   accept: {
     type: String,
     default: '',
   },
+  /**
+   * @description
+   */
```

Comment:
`type` also seems to be a useless attribute. 🤔

Reply:
updated, I have deleted it.